### PR TITLE
Coutoire: Menu Enhancements

### PR DESCRIPTION
**Pull request by scruffian**
This is an alternative to https://github.com/Automattic/themes/pull/4041, which I encountered a few issues with. (I couldn't put to that PR as it was created in a fork).

The main things I noticed were:
1. When you scrolled down the content would jump up before animating back into position
2. Sometimes when the page loaded the site logo animation would fire unexpectedly
3. The first time the site logo animation fired it was often jumpy.

co-authored with @Aurorum

### CODE REVIEW

File: coutoire/js/fixed-header-spacing.js [hunk 1]
```diff
@@ -42,46 +41,88 @@
...
+
 	/**
 	 * Run our page header height function
 	 */
 	pageHeaderHeight();
-})();
+
+	setTimeout( function () {
+		window.addEventListener( 'scroll', logoWidth );
+	}, 26 ); // We need to wait for the margin top to be set before we attach the scroll handler.
```
Comment (scruffian) [inline coutoire/js/fixed-header-spacing.js:127]: This is really nasty, but I found that sometimes the scroll event was firing when the page loaded, and before the other things above had finished running.

File: coutoire/sass/_extra-child-theme.scss [hunk 2]
```diff
@@ -385,4 +385,27 @@ a.wp-block-file__button {
...
+
+/**
+ * Animate logo resizing.
+ */
+ .site-logo {
+	width: 100%;
+	-webkit-transition: 0.25s linear;
+	-moz-transition: 0.25s linear;
+	-o-transition: 0.25s linear;
+	-ms-transition: 0.25s linear;
```
Comment (MaggieCabrera) [inline coutoire/sass/_extra-child-theme.scss:398]: super minor nitpick but I think the prefixes are added automatically during the build process

**Comment by MaggieCabrera** [comment]
This looks like it's working fine. The only thing I see is that the tagline/site title can overlap the content if it's too long (and the font size is quite big, so it's bound to happen):

Screenshot 2021-10-27 at 11 30 50

**Comment by kjellr** [comment]
(I am using a laughably huge image here, but) I'm seeing some weird double-animation when I scroll back up to the top of the page in Safari: 

scrolling


This doesn't happen in Chrome.

**Comment by scruffian** [comment]
@jeffikus do you think this is still worth us investing time in?

**Comment by scruffian** [comment]
We're putting this on hold for now

**Pull request closed without merging**